Sparkle/SUTouchBar.m

```diff
@@ -0,0 +1,94 @@
+//
+//  SUTouchBar.m
+//  Sparkle
+//
+//  Created by Yuxin Wang on 05/01/2017.
+//  Copyright © 2017 Sparkle Project. All rights reserved.
+//
+
+#import "SUTouchBar.h"
+
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101202
```

Comment:
Yes, it is a SDK version check.
If SDK version is lower than 10.12.2, there are not definitions for NSTouchBar and it's related APIs. So I added fake implementation to make sure the file can be compiled
That means, if Sparkle compile with SDK < 10.12.2, there will be no Touch Bar support.